meshtastic/interfaces/ble/lifecycle_ownership_runtime.py

```
@@ -0,0 +1,432 @@
+        if is_closing:
+            logger.debug(
+                _LOG_INTERFACE_CLOSED_DURING_CONNECT,
+                getattr(connected_client, "address", "unknown"),
+            )
+            return
+        logger.debug(
+            _LOG_INTERFACE_LOST_OWNERSHIP_DURING_CONNECT,
+            getattr(connected_client, "address", "unknown"),
+        )
+
+    def _discard_invalidated_connected_client(
```

Ruff (0.15.6)

[warning] Too many statements (58 > 50)

(PLR0915)